src/main/java/nl/tudelft/lifetiles/notification/controller/NotificationController.java

```diff
@@ -0,0 +1,131 @@
+package nl.tudelft.lifetiles.notification.controller;
+
+import java.net.URL;
+import java.util.PriorityQueue;
+import java.util.Queue;
+import java.util.ResourceBundle;
+
+import javafx.animation.KeyFrame;
+import javafx.animation.Timeline;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.util.Duration;
+import nl.tudelft.lifetiles.core.controller.Controller;
+import nl.tudelft.lifetiles.core.util.ColorUtils;
+import nl.tudelft.lifetiles.core.util.Message;
+import nl.tudelft.lifetiles.notification.model.Notification;
+
+/**
+ * The controller for the notification view.
+ *
+ * @author Joren Hammudoglu
+ *
+ */
+public class NotificationController extends Controller {
+
+    /**
+     * Notification shout message.
+     */
+    public static final Message NOTIFY = Message.create("notification");
+
+    /**
+     * The wrapper.
+     */
+    @FXML
+    private HBox wrapper;
+
+    /**
+     * The text field.
+     */
+    @FXML
+    private Label label;
+
+    /**
+     * The close label.
+     */
+    @FXML
+    private Button close;
+
+    /**
+     * The notifications to display.
+     */
+    private Queue<Notification> notifications;
+
+    /**
+     * A notification is displaying.
+     */
+    private boolean displaying;
+
+    @Override
+    public final void initialize(final URL location,
+            final ResourceBundle resources) {
+        final int initialCapacity = 10;
+        this.notifications = new PriorityQueue<>(initialCapacity,
+                (n1, n2) -> n1.getPriority() - n2.getPriority());
```

Comment:
Comparable needs to be consistent with equals. It should return 0 iff equals returns true (and hashmap returns the same value on both objects)